src/Microsoft.Diagnostics.Monitoring.Extension.Common/EgressHelper.cs

```diff
@@ -129,7 +129,9 @@ private static int ProcessEgressResult(EgressArtifactResult result)
             // dependency injection.
             services.TryAddSingleton<EgressProvider<TOptions>, TProvider>();
 
+#if NET7_0_OR_GREATER
```

Comment:
I take it this is a result of downgrading from 7/8 to 6 correct?

Reply:
Yes